table/sst_file_dumper.cc

```diff
@@ -187,7 +190,11 @@ uint64_t SstFileDumper::CalculateCompressedTableSize(
     uint64_t* num_data_blocks) {
   std::unique_ptr<WritableFile> out_file;
   std::unique_ptr<Env> env(NewMemEnv(options_.env));
-  env->NewWritableFile(testFileName, &out_file, soptions_);
+  Status s = env->NewWritableFile(testFileName, &out_file, soptions_);
+  if (!s.ok()) {
+    fputs(s.ToString().c_str(), stderr);
+    exit(1);
```

Comment:
I see this function already has some calls to exit, but this makes me nervous as we recently added a call from library code to SstFileDumper (in backupable_db.cc). Can we go ahead and refactor this to return status? 🙏 🙏

Reply:
Updated.